src/streaming/controllers/ProtectionController.js

```diff
@@ -197,7 +197,7 @@ MediaPlayer.dependencies.ProtectionController = function () {
             this.protectionModel.init();
         },
 
-        init: function (manifest) {
+        init: function (aInfo, vInfo) {
```

Comment:
@KozhinM This is going to break the eme.html demo page (eme-main.js).  Unfortunately, this changes the API for ProtectionController (which is used outside the player for pre-fetching licenses) and there is no way for the app to get at the streamInfo objects that init() now needs.